src/util/system.cpp

```diff
@@ -894,14 +894,19 @@ bool ArgsManager::ReadConfigFiles(std::string& error, bool ignore_invalid_keys)
         if (!ReadConfigStream(stream, confPath, error, ignore_invalid_keys)) {
             return false;
         }
-        // if there is an -includeconf in the override args, but it is empty, that means the user
-        // passed '-noincludeconf' on the command line, in which case we should not include anything
-        bool emptyIncludeConf;
+        // `-includeconf` cannot be included in the command line arguments except
+        // as `-noincludeconf` (which indicates that no conf file should be used).
```

Comment:
I find comment a bit confusing, `-noincludeconf` means we are only going to discard `includeconf` in config file but not the main config itself (at least I've tested that the current behavior with this patchset)

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15934#discussion_r344270169

> I find comment a bit confusing, `-noincludeconf` means we are only going to discard `includeconf` in config file but not the main config itself (at least I've tested that the current behavior with this patchset)

`-noincludeconf` just disables the `-includeconf` directive. I wouldn't expect it to disable the whole config file. (I might expect `-noconf` to do that.) So this doesn't seem that confusing to me, but I'm happy to take a suggestion if you have an idea to improve it.

The current text just comes from John's suggestion https://github.com/bitcoin/bitcoin/pull/15934#discussion_r343795528

Reply:
nit-picking there, was just to change comment to `(which indicates that no included conf file should be used)`